### ISSUE TITLE
Model Browser Not loading the Constraints as Shown in Fig 2-19 and 2-20

### ISSUE THREAD

**Issue by goshi-f-a** [dresden-ocl/dresdenocl#70]
I am going through the Manual Steps. 
Using LUNA with Java Version 1.8.0_51.
Using Dresden ver 3.4.1 from http://dresden-ocl.org/update/luna/latest
Updates already done from http://www.emftext.org/update/ and http://download.eclipse.org/tools/ajdt/42/dev/update/ 

Now at step 2.2.5 Parsing OCL Expressions:
Already loaded model (SimpleExample) as well as model instance. But when I open the constraints file in OCL Editor (which is EMFText OCL Editor and not Dresden OCL Editor), the constraints are supposed to be loaded in  Model Editor but these are not being loaded. In fact Dresden OCL Editor option is not visible. Screenshot is below
Please Help...
![image](https://cloud.githubusercontent.com/assets/13867157/9377340/6a466ce2-4731-11e5-8d8a-b02331a770be.png)

**Comment by lschuetze** [comment]
Could you please test to use Java 6 (or try Java7) and run your Eclipse with this version? Therefor you edit the `eclipse.ini` and add at the very first line:

```
-vm
C:\Program Files\Java\jdk1.6.0_45\bin\javaw.exe
```

or however your path to your JDK 1.6 is. Please report back. I just tested and I can see the constraints in the model browser.
![image](https://cloud.githubusercontent.com/assets/2911272/9380229/2c223632-472f-11e5-8e4a-b459bb81d185.png)

**Comment by goshi-f-a** [comment]
After installing JDK 6 as well as 7 and path setting in eclipse.ini. Model is loaded but Model Instance loading gives following error. One more symptom I am getting that context menu in Package Explorer is not working in Dredsen Perspective but working in java perspective.
![image](https://cloud.githubusercontent.com/assets/13867157/9390329/f789931a-4789-11e5-8c4e-66bc9ce5046c.png)
Just a while ago I have updated the Dredsen project after receiving update notification. 
After reverting back to original eclipse.ini. Now EMFTOCLeditor is not opening the constraint file. See following screen shot
![image](https://cloud.githubusercontent.com/assets/13867157/9391878/ef41ff7c-4792-11e5-8d78-c60309feda56.png)
Please guide: should I download fresh LUNA, if yes then which release? There are so many of these here: https://eclipse.org/downloads/packages/all

**Comment by lschuetze** [comment]
I think Eclipse is still just setting your JDK8 everywhere. If you're looking under `Window > Preferences > Java > Installed JREs` there have to be your JDK6 & 7, too. I think there is just JDK8 and its set for every JDK in the `Execution Environment` part. `major.minor version 52` is Java 8.

If you're going to download a [new Eclipse](https://eclipse.org/downloads/packages/release/Luna/SR2) I would suggest either the [Modeling Tools](https://eclipse.org/downloads/packages/eclipse-modeling-tools/lunasr2) or the [RCP and RAP](https://eclipse.org/downloads/packages/eclipse-rcp-and-rap-developers/lunasr2) one.

**Comment by goshi-f-a** [comment]
Your assessment was right about Installed JREs. Only JDK 8 was there. Now I have downloaded fresh LUNA (Modeling Tools). Added JRE 6 and 7 and made each default one by one as shown below. (I think if I set the default here, I don't require to make changes in eclipse.ini. Am I right?)
![image](https://cloud.githubusercontent.com/assets/13867157/9421590/81038ca8-488f-11e5-8ee3-a6b69c2c9eb7.png)
Anyway I am able to load the constraints in one scenario, that is, Load the example fully -> save the project -> restart eclipse. Same is the case for JRE 6, 7 and 8.......After restarting the screenshot is below. 
![image](https://cloud.githubusercontent.com/assets/13867157/9421626/53140916-4890-11e5-99aa-214ab9c7213f.png)
 After reaching here I have the issue in next step of the tutorial step, which I will put in other Thread.
